### ISSUE TITLE
Ability to specify data outside the render() function

### ISSUE THREAD

**Issue by sivann** [slimphp/PHP-View#21]
It would be very helpful to have a variable from request ($request->getUri()->getBasePath();) always available on the templates e.g. with a middleware without having to pass the above as parameter to renderer->render on all routes each time (like so):
```
$app->get(...
   ...
  $args['basepath']=$request->getUri()->getBasePath();
  return $this->renderer->render($response, 'test.php', $args);
});
```

**Comment by geggleto** [comment]
Ah yes, I totally forgot about the base path.

**Comment by sivann** [comment]
That response time is legendary :-D

**Comment by geggleto** [comment]
I get notifications in IRC about issues ;)

**Comment by akrabat** [comment]
Fixed by #22
